accounts/forms.py

```diff
@@ -149,4 +149,12 @@ def assign(self, profile):
         profile.firstname = self.cleaned_data['firstname']
         profile.lastname = self.cleaned_data['lastname']
         profile.sex = self.cleaned_data['sex']
-        profile.phone = self.cleaned_data['phone']
\ No newline at end of file
+        profile.phone = self.cleaned_data['phone']
+
+
+class SIGForm(BasicForm):
+    SigMain1 = forms.ChoiceField(choices=Status.SIG_TYPES_MAIN)
```

Comment:
`setup_field()` is also required to be added for proper bootstrap view and all, check how other forms are made

Reply:
Formatted properly now.